config.core-dev.yaml

```diff
@@ -0,0 +1,10 @@
+# #ddev-generated
+# This file is placed by the justafish/ddev-drupal-core-dev addon.
+
+docroot: ""
+php_version: "8.3"
```

Comment:
could be removed it is already auto set in the config.yaml

Reply:
maybe that is the only one that might make sense on second thought. for folks not on 1.23.0 yet. that way it is made sure the project runs on php 8.3 which is required for drupal 11